user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -162,7 +162,7 @@ private void addMeasurementTab(GridLazyDataView<?> gridLazyDataView) {
   private void createNGSMeasurementGrid() {
     ngsMeasurementGrid.addClassName("measurement-grid");
     ngsMeasurementGrid.addColumn(ngsMeasurement -> ngsMeasurement.measurementCode().value())
-        .setHeader("Measurement Code");
+        .setHeader("Measurement ID");
     ngsMeasurementGrid.addComponentColumn(
             ngsMeasurement -> renderSampleCodes().createComponent(ngsMeasurement.measuredSamples()))
         .setHeader("Sample Codes");
```

Comment:
Please also adjust the column headers of the sample code column for ngs and proteomics